.actor/dataset_schema.json

```diff
@@ -0,0 +1,68 @@
+{
+    "actorSpecification": 1,
+    "fields": {},
+    "views": {
+        "overview": {
+            "title": "Overview",
+            "transformation": {
+                "fields": [
+                    "company",
+                    "mode",
+                    "departureTime",
+                    "arrivalTime",
+                    "duration",
+                    "departure",
+                    "arrival",
+                    "price",
+                    "ticketsLeft",
+                    "stops"
+                ]
+            },
+            "display": {
+                "component": "table",
+                "properties": {
+                    "company": {
+                        "label": "Company",
+                        "format": "text"
+                    },
+                    "mode": {
+                        "label": "Mode",
+                        "format": "text"
+                    },
+                    "departureTime": {
+                        "label": "Departure Time",
+                        "format": "datetime"
+                    },
+                    "arrivalTime": {
+                        "label": "Arrival Time",
+                        "format": "datetime"
+                    },
+                    "duration": {
+                        "label": "Duration",
+                        "format": "text"
+                    },
+                    "departure": {
+                        "label": "Departure Location",
+                        "format": "text"
+                    },
+                    "arrival": {
+                        "label": "Arrival Location",
+                        "format": "text"
+                    },
+                    "price": {
```

Comment:
there should be a `currency` column next to the price. In this [run](https://console.apify.com/actors/CRHMKzgdNtmb91G1r/runs/T3lnBFslhZdt0m6UZ#output), the price from brno to prague is `23428`, which doesn't seem to be CZK :thinking: 

![image](https://github.com/user-attachments/assets/4601b9b3-869e-4825-968c-abdc2b25d6a6)